assets/css/components/_app-toast.scss

```diff
@@ -0,0 +1,23 @@
+.app-toast {
+  position: absolute;
+  margin: auto;
+  left: 0;
+  right: 0;
+  max-width: rem(500px);
+
+  &__alert {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__message {
+    margin: 0;
+  }
+
+  &__close {
```

Comment:
The current name is a bit abstract, let's rename it to be more component maybe `&__close-btn` or  `&__dismiss-btn`

Reply:
Added in [9bcc594](https://github.com/rafayet-monon/elixir_search_extractor/pull/29/commits/9bcc594b71e75142d8612d62a008febd413e6c66)